api-server/test/fakers/event.faker.ts

```diff
@@ -132,4 +132,65 @@ export async function createEvent(
   return eventCreated;
 }
 
+export async function createListOfFutureEventsInChronologicalOrder(
+  eventModel: typeof EventModel,
+  venueModel: typeof VenueModel,
+  datetimeVenueModel: typeof DatetimeVenueModel,
+  numEvents = 30,
+  overrides: EventModelConstructorProps = {},
+  baseTime = new Date(),
+): Promise<[EventModel[], Date]> {
+  const events: EventModel[] = [];
+
+  for (let i = 0; i < numEvents; i++) {
+    const [newEvent, newBaseTime] = await createEventInPast(
```

Comment:
It's a little confusing that `createListof**Future**EventsInChronologicalOrder` delegates to another helper called `createEvent**InPast**`.

Reply:
ah, indeed lol, probably need to rethink those names